src/main/kotlin/frc/robot/subsystems/DrivetrainSubsystem.kt

```diff
@@ -9,8 +9,10 @@ import com.kauailabs.navx.frc.AHRS
  * TODO: add gyro and encoder handling
  */
 class DrivetrainSubsystem(val drivetrain: DifferentialDrive, val gyroscope: AHRS) : SubsystemBase() {
+    // Accepts the drivetrain type (tank), and the gyroscope type.
     /* default command (don't drive) */
     override fun periodic() {
+        // Sets both sides to 0.
```

Comment:
This comment is unnecessary. It is clear to anybody reading the code that both sides are set to 0. The `/* default command (don't drive) */` comment is a better example of a comment (though not great). It describes what the code is meant to achieve, not exactly how it does it (which the code itself describes).